WebRoot/js/zimbraMail/mail/controller/ZmComposeController.js

```diff
@@ -1228,7 +1228,7 @@ function() {
 		buttons.push(ZmOperation.SEND);
 	}
 
-	buttons.push(ZmOperation.CANCEL, ZmOperation.SEP, ZmOperation.SAVE_DRAFT);
+	buttons.push(ZmOperation.SAVE_DRAFT, ZmOperation.SEP, ZmOperation.CANCEL);
```

Comment:
ZmOperation.SEP is creating a bit of vertical spacing between the two labels, here. It can be removed as it is suppressing the spacing between two labels to small extent.
But, there are many other places in the same view , where ZmOperation.SEP is being used and I didn't removed them to be consistent.